config.py

```diff
@@ -25,7 +25,16 @@
 shp_fp = OUTPUT_DIR.joinpath("detected_ars.shp")
 
 # path for shapefile column name crosswalk csv
-csv_fp = OUTPUT_DIR.joinpath("columns.csv")
+csv_fp = OUTPUT_DIR.joinpath("detected_ars_columns.csv")
+
+# path for landfall ARs (6hr) results shapefile
+landfall_6hr_fp = OUTPUT_DIR.joinpath("landfall_ars_6hr.shp")
+
+# path for landfall ARs (events) results shapefile
+landfall_events_fp = OUTPUT_DIR.joinpath("landfall_ars_events.shp")
+
+# path to Alaska coastline shapefile (included in Github repo tree)
+ak_shp = Path('./shp/Alaska_Coast_Simplified_Polygon.shp')
```

Comment:
```suggestion
ak_shp = Path("./shp/Alaska_Coast_Simplified_Polygon.shp")
```
Convention is to use double quotes `"` to encapsulate strings. This is something that the Black formatter usually will catch and change automatically, but developing on the remote it seems like our formatter is success is mixed. If you want, you should be able to commit this suggestion straight into the branch!